DependencyInjection/FOSRestExtension.php

```diff
@@ -362,6 +362,10 @@ private function loadSerializer(array $config, ContainerBuilder $container): voi
 
         $options['serializeNullStrategy'] = $config['serializer']['serialize_null'];
         $viewHandler->addArgument($options);
+
+        if ($config['serializer']['disable_custom_jms_registry']) {
```

Comment:
I'm not sure whether we should trigger a deprecation warning in case this option is not activated as it will create noise for the user but shouldn't make the removal of the custom registry safer as the deprecation triggered [here](https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2227/files#diff-87bb7b48eab03df25a3b87a9dd7e44ccR61) ensures it has no effect.